recipes/paramiko/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "paramiko" %}
+{% set version = "2.0.2" %}
+{% set checksum = "411bf90fa22b078a923ff19ef9772c1115a0953702db93549a2848acefd141dc" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cryptography >=1.1
+    - pyasn1 >=0.1.7
+
+  run:
+    - python
+    - cryptography >=1.1
+    - pyasn1 >=0.1.7
+
+test:
+  imports:
+    - paramiko
+
+about:
+  home: https://github.com/paramiko/paramiko/
+  license: LGPL 2.1
+  summary: SSH2 protocol library
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
@danielfrg, @koverholt, @kwilcox, @ocefpaf, @vamega, it seems all of you have packages that depend on `paramiko` that you are maintaining. Do any of you have an interest in helping me maintain `paramiko`? This should help make sure it is in good shape for you to use elsewhere.